### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by kyle1morel**
# Description

Formal release PR for PCNS v0.3.0.

https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-170

## Types of changes

Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+  };
+});
+
+async function emailConfirmation(activityId: string) {
+  const configCC = getConfig.value.ches?.submission?.cc;
+  const body = confirmationTemplate({
+    '{{ contactName }}': formRef.value?.values.applicant.contactFirstName,
+    '{{ activityId }}': activityId
+  });
+  let applicantEmail = formRef.value?.values.applicant.email;
```
Comment (sanjaytkbabu) [inline frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue:225]: please update to contactEmail

**Comment by sanjaytkbabu** [comment]
As discussed FRONTEND_CHES_BCC need to be changed to FRONTEND_CHES_ROADMAP_BCC in nr-permitconnect-navigator-service-config to align with line #110 in RoadMap.vue.

**Comment by slhurley** [comment]
First....there so many good things about this!  But what you want to know:
- Will Developer tab be hidden by default unless it is us?
- Last Edited Date is not shown on the submission list until after you get past the first tab (save only first tab, view draft, no date)
- We are missing an "s" in the word successfully on the confirmation page after submitting.
- We need a "coming soon" when you click the Status button before we could release this to proponents.
- The Files tab seems broken...list view is not working and I believe a file I uploaded disappeared on subsequent login.
- If you click on the BF note from the notification bar or from the BF calendar, you get a 422.
- There is no search option on the Enquiry list view 
- We were hoping to have Address and Business lookups included on the Navigator view of submissions
- If a Navigator creates a submission in their view and enters an address it is still forcing them to enter lat/long.  It shouldn't.
- The Assigned To field is acting odd.  It doesn't do a dropdown and if you want to switch people you have to do a bunch of backspacing first and then enter a new set of 3 characters.

**Comment by kyle1morel** [comment]
> First....there so many good things about this! But what you want to know:
> 
> * Will Developer tab be hidden by default unless it is us?
> * Last Edited Date is not shown on the submission list until after you get past the first tab (save only first tab, view draft, no date)
> * We are missing an "s" in the word successfully on the confirmation page after submitting.
> * We need a "coming soon" when you click the Status button before we could release this to proponents.
> * The Files tab seems broken...list view is not working and I believe a file I uploaded disappeared on subsequent login.
> * If you click on the BF note from the notification bar or from the BF calendar, you get a 422.
> * There is no search option on the Enquiry list view
> * We were hoping to have Address and Business lookups included on the Navigator view of submissions
> * If a Navigator creates a submission in their view and enters an address it is still forcing them to enter lat/long.  It shouldn't.
> * The Assigned To field is acting odd.  It doesn't do a dropdown and if you want to switch people you have to do a bunch of backspacing first and then enter a new set of 3 characters.

The developer tab is only visible to those with the developer role. Nobody outside the dev team will ever see, or be able to access it.

**Review by wilwong89 — APPROVED** [review]

**Review by sanjaytkbabu — APPROVED** [review]

**Comment by kyle1morel** [comment]
> First....there so many good things about this! But what you want to know:
> 
> * Will Developer tab be hidden by default unless it is us?
> * Last Edited Date is not shown on the submission list until after you get past the first tab (save only first tab, view draft, no date)
> * We are missing an "s" in the word successfully on the confirmation page after submitting.
> * We need a "coming soon" when you click the Status button before we could release this to proponents.
> * The Files tab seems broken...list view is not working and I believe a file I uploaded disappeared on subsequent login.
> * If you click on the BF note from the notification bar or from the BF calendar, you get a 422.
> * There is no search option on the Enquiry list view
> * We were hoping to have Address and Business lookups included on the Navigator view of submissions
> * If a Navigator creates a submission in their view and enters an address it is still forcing them to enter lat/long.  It shouldn't.
> * The Assigned To field is acting odd.  It doesn't do a dropdown and if you want to switch people you have to do a bunch of backspacing first and then enter a new set of 3 characters.

Resolved most noted issues. After discussion, we will not be making adjustments to the Assigned To dropdown, or adding the Address and Business lookups to the Nav submission view at this time.

**Pull request merged**